lib/get-config.js

```diff
@@ -31,7 +32,7 @@ async function requirePlugin(workingDir, pluginName, fromConfigPath) {
   let pluginPath = resolve.sync(pluginName, { basedir, extensions: ['.js', '.mjs', '.cjs'] });
```

Comment:
Could the reason we end up needing this be related to `resolve.sync` usage here? I wonder if we can solve this by removing the `resolve.sync` somehow instead? 

Although this link shows your similar workaround: https://github.com/nodejs/help/issues/3380

Also, I'm still hoping we can figure out actual tests for this that fail beforehand.

Reply:
@bmish you mean change this logic in scope of this PR? (it's already in codebase)